console/console-init/ui/src/modules/project/components/ProjectDatatoolbarToggleGroup/ProjectDataToolbarToggleGroup.tsx

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  SelectOptionObject,
+  DataToolbarToggleGroup,
+  DataToolbarGroup,
+  DataToolbarFilter,
+  InputGroup,
+  Button,
+  DataToolbarItem,
+  ButtonVariant,
+  DataToolbarChip,
+  DataToolbarChipGroup,
+  DropdownPosition,
+  Badge
+} from "@patternfly/react-core";
+import { ISelectOption } from "utils";
+import { FilterIcon, SearchIcon } from "@patternfly/react-icons";
+import {
+  DropdownWithToggle,
+  TypeAheadSelect,
+  DropdownWithBulkSelect
+} from "components";
+
+export interface IProjectToolbarToggleGroupProps {
+  totalRecords: number;
+  filterSelected?: string;
+  nameSelected?: string;
+  nameInput?: string;
+  namespaceSelected?: string;
+  namespaceInput?: string;
+  nameOptions?: any[];
+  namespaceOptions?: any[];
+  typeSelected?: string | null;
+  selectedNames: Array<{ value: string; isExact: boolean }>;
+  selectedNamespaces: Array<{ value: string; isExact: boolean }>;
+  onFilterSelect: (value: string) => void;
+  onNameSelect: (e: any, selection: SelectOptionObject) => void;
+  onNameClear: () => void;
+  onNamespaceSelect: (e: any, selection: SelectOptionObject) => void;
+  onNamespaceClear: () => void;
+  onTypeSelect: (selection: string) => void;
+  onDeleteAll: () => void;
+  onSearch: () => void;
+  onDelete: (
+    category: string | DataToolbarChipGroup,
+    chip: string | DataToolbarChip
+  ) => void;
+  onChangeNameInput?: (value: string) => Promise<any>;
+  onChangeNameSpaceInput?: (value: string) => Promise<any>;
+  setNameInput?: (value: string) => void;
+  setNameSpaceInput?: (value: string) => void;
+  isAllProjectSelected: boolean;
+  onSelectAllProjects: (val: boolean) => void;
+}
+const ProjectToolbarToggleGroup: React.FunctionComponent<IProjectToolbarToggleGroupProps> = ({
+  totalRecords,
+  filterSelected,
+  nameSelected,
+  nameInput,
+  namespaceSelected,
+  namespaceInput,
+  typeSelected,
+  selectedNames,
+  selectedNamespaces,
+  onFilterSelect,
+  onNameSelect,
+  onNameClear,
+  onNamespaceSelect,
+  onNamespaceClear,
+  onTypeSelect,
+  onSearch,
+  onDelete,
+  onChangeNameInput,
+  onChangeNameSpaceInput,
+  setNameInput,
+  setNameSpaceInput,
+  isAllProjectSelected,
+  onSelectAllProjects
+}) => {
+  const filterMenuItems = [
```

Comment:
As per the discussed guidelines, all dropdown options need to be in a generic file, in this case in Utils.